canonical-webteam/documentation-builder
# Add default homepage template

## Don't merge until #110 is merged

## Done

- Added a default homepage template to the documentation-builder docs, as a base for building a homepage similar to docs.vanillaframework.io
- Added the page to `metadata.yaml`

## QA

- Copy `ubuntudesign/documentation_builder/resources/template.html` into the `docs/` folder
- Edit `docs/package.json` so it uses this template
``` json
"build": "documentation-builder --base-directory .  --output-media-path 'static/media' --media-url '/static/media' --output-path templates --tag-manager-code 'GTM-K92JCQ' --no-link-extensions --force --search-url 'https://docs.ubuntu.com/search' --search-placeholder 'Search Builder docs' --template-path template.html",
```
- Edit `docs/requirements.txt` and version of `ubuntudesign.documentation-builder` to `1.5.1` (this is a bug - 1.5.3 hasn't been released yet?)
- `./run` while in `docs/` folder
- Open http://0.0.0.0:8000/en/homepage-template
- Check that it matches the original [design](https://github.com/ubuntudesign/vanilla-design/issues/217#issuecomment-384349194)

## Screenshots

![0 0 0 0_8000_en_homepage-template 2](https://user-images.githubusercontent.com/25733845/41351284-cc14a69e-6f0d-11e8-912f-ce05e0234793.png)
